internal/stokerhttp/get.go

```diff
@@ -1,48 +1,66 @@
 package stokerhttp
 
 import (
-	"context"
 	"errors"
 	"fmt"
 	"net/http"
-	"net/url"
 	"strconv"
 	"time"
 
-	"github.com/frantjc/sindri/internal/appinfoutil"
 	"github.com/frantjc/sindri/steamapp/postgres"
 	"github.com/go-chi/chi"
 	"github.com/go-logr/logr"
 )
 
-type SteamappMetadata struct {
-	AppID       int       `json:"app_id"`
-	Name        string    `json:"name,omitempty"`
-	IconURL     string    `json:"icon_url,omitempty"`
-	DateCreated time.Time `json:"date_created"`
-	DateUpdated time.Time `json:"date_updated"`
-	Locked      bool      `json:"locked,omitempty"`
+type SteamappList struct {
+	Offset    int        `json:"offset"`
+	Limit     int        `json:"limit"`
+	Steamapps []Steamapp `json:"steamapps"`
+}
+
+type Steamapp struct {
```

Comment:
I think we should add AppID on this struct directly

Reply:
my thinking was:

```go
// The stuff that gets returned from the list endpoint--needs to incl AppID
type SteamappInfo

// The stuff that a user would send to us
type SteamappSpec

// The above two zipped together
type Steamapp
```

Reply:
I was suggesting moving app ID up to the `Steamapp` struct because it pertains to both of the other structs. `SteamappInfo` is what comes from `steamcmd` and `SteamappSpec` is what comes from the user. We use app ID to accept/get information from both of those

Reply:
There are other things I don't like about getting `SteamappSpec` from the user. For example, `date_created` & `date_updated` we set at time of database insert, so it makes no sense to accept them from the user.

Reply:
Also `locked` is always set to false right now. I was wondering if that would be an internal thing for us and not passable by the user